src/styles/Radio.scss

```diff
@@ -1,41 +1,52 @@
 .radio-selector {
-    display: flex;
-    height: 60px;
-    font-size: 1rem;
+  display: flex;
+  height: 60px;
+  font-size: 1rem;
 }
 
 .radio-option {
-    border: 2px solid #ddd;
-    background-color: rgb(238, 170, 25);
-    color: #eee;
-    padding: 5px;
-    cursor: pointer;
-    line-height: 27.5px;
+  //   border: 2px solid #ddd;
+  background-color: #473d5d;
+  color: #eee;
+  padding: 8px;
+  cursor: pointer;
+  max-width: 70px;
 }
 
 .radio-option:hover {
-    box-shadow: inset 0 0 0 50px rgba(0, 0, 0, 0.2);
+  box-shadow: inset 0 0 0 50px rgba(0, 0, 0, 0.2);
 }
 
 .radio-option-selected {
-    border: 2px solid #ddd;
-    background-color: #4caf50;
-    color: #eee;
-    padding: 5px;
-    cursor: pointer;
-    line-height: 27.5px;
+  border: none !important;
+  background-color: #7a45f0;
```

Comment:
Not a required change, but I think it might be good if we could change this to a SASS variable. Eventually we'll be writing some custom themes so variables make our life easier!